enterprise/models.py

```diff
@@ -1864,6 +1864,11 @@ class EnterpriseCatalogQuery(TimeStampedModel):
         ),
         validators=[validate_content_filter_fields]
     )
+    uuid = models.UUIDField(
+        unique=True,
+        default=uuid4,
+        editable=False,
```

Comment:
Consider making `editable=True` and having it be a read-only field in the admin site, so that it's easy to find a UUID for pasting into config files without having to go to the read-replica.

Reply:
ooo fair point.